adapters/handlers/rest/dynamic_user/handlers_dynamic_users.go

```diff
@@ -231,6 +233,72 @@ func (h *dynUserHandler) deleteUser(params users.DeleteUserParams, principal *mo
 	return users.NewDeleteUserNoContent()
 }
 
+func (h *dynUserHandler) suspendUser(params users.SuspendUserParams, principal *models.Principal) middleware.Responder {
```

Comment:
Not sure about you, but `suspend` to me is not the direct opposite of `activate`. wdyt about `deactivate/activate` or was `suspend` specifically desired?

Reply:
I like your suggestion.
I meant suspend as a wider goal, something that I might want to do when managing my users.
But I agree that deactivate is a better name for it in the API.

This way, I can deactivate->activate users to suspend and enable them back again.

Reply:
changed :)